### PR TITLE
Prevent exception when encountering a node of unknown type

### PR DESCRIPTION
**Pull request by nicolaslt**
@michaelficarra This is a reopen of #71. I had deleted my fork so couldn't fix the original PR :disappointed:

### CODE REVIEW

File: tests/unknownNodeType.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  "./fixtures/unknownNodeTypeAST",
+], function (esquery, assert, test, AST) {
+
+  test.defineSuite("Unknown node type", {
+    "does not throw": function () {
+      var thrown = false
+      try {
+        esquery(AST, '*');
+      } catch (e) {
+        thrown = true
```
Comment (nicolaslt) [inline tests/unknownNodeType.js]: Original comment from @michaelficarra 

> Just `assert.fail()` here.

Do you have documentation on assert.fail? I couldn't find it in `jstestr`'s doc.

**Comment by okunokentaro** [comment]
I am the same. I want to fix it. 👍

**Comment by michaelficarra** [comment]
@nicolaslt can you check the box to allow edits from maintainers over on the right panel? It should look like this:

image

**Comment by michaelficarra** [comment]
Never mind, I've merged as https://github.com/estools/esquery/commit/d48682f41c6b6a462ce56a5b3e08c124e5997533. Thanks for the contribution.

**Comment by nicolaslt** [comment]
Awesome, thanks!

**Comment by michaelficarra** [comment]
Btw I released `1.1.0` and @nicolaslt you should add the email address you used to make [the commit](https://github.com/estools/esquery/commit/d48682f41c6b6a462ce56a5b3e08c124e5997533.patch) to your GitHub emails if you want to have the commit properly attributed to your account.

**Pull request closed without merging**